tasks/main-blivet.yml

```diff
@@ -52,7 +52,7 @@
 - name: Apply defaults to pools and volumes [4/6]
   set_fact:
     _storage_vols_no_defaults_by_pool: "{{ _storage_vols_no_defaults_by_pool|default({})|
-                                        combine({item.name: _storage_vols_w_defaults|selectattr('pool', 'equalto', item.name)|list}) }}"
+                                        combine({item.name: _storage_vols_w_defaults|json_query('[?pool==`{}`]'.format(item.name))}) }}"
```

Comment:
what is the meaning of the backticks (`` `{}` ``) ?

Reply:
AIUI, backticks are just another quotation marks.
`{}` is a placeholder for format

Reply:
> AIUI, backticks are just another quotation marks.

Is it documented anywhere? I checked Jinja docs and they do not mention anything about backticks.

Reply:
Hmm… after reading specs, backticks are used to enclose JSON value… I should use either `` `"string"` `` or `'string'`

Reply:
@yontalcar  Can you link the specs you are using please?

Reply:
it's JMESPath thing: http://jmespath.org/specification.html